test/unit/package_test.rb

```diff
@@ -36,6 +36,17 @@ class PackageTest < Minitest::Test
       assert_equal("components/timeline/my/path/", package.public_path)
     end
 
+    test "#public_path returns expected path when using the default public path in root package" do
+      package = Package.new(name: ".", config: {})
+      assert_equal("./app/public/", package.public_path)
```

Comment:
I don't think the public path starting with `.` is expect or even correct.

Reply:
I've fixed up the test coverage that I added and nothing else appears to break with the suggested changes.